src/App.vue

```diff
@@ -71,18 +71,36 @@ export default Vue.extend({
     };
 
     /**
-     * Keep connection to NEST Server alive.
+     * Update configs from global config.
+     *
+     * @remarks
+     * Global config is loaded in main.ts.
+     */
+    const updateConfigs = () => {
+      const config = Vue.prototype.$config || {};
+
+      // Update config for NEST Simulator
+      if (config.NESTSimulator && !core.app.NESTSimulator.config.custom) {
+        if ('url' in config.NESTSimulator) {
+          core.app.NESTSimulator.url = config.NESTSimulator.url;
+        } else {
+          core.app.NESTSimulator.updateConfig(config.NESTSimulator);
+        }
+      }
+    };
```

Comment:
I place the method here because the core app should be initialized before.

I thought the function is adequate. 
The first condition checks the existence of NESTSimulator in config and when URL is not customized by the user.
The second condition checks when the the config contains URL, e.g. `https://nest-desktop.app.hbp.eu/nest `
Otherwise it only updates port and host if existed in $config.

Reply:
My bad, sorry! :see_no_evil:  `app.ts` would be the class where it should be placed, not `main.ts`. I added the suggestion in a PR in your fork.

Reply:
Comment resolved in https://github.com/babsey/nest-desktop/pull/3